### ISSUE TITLE
SCSS resouce file possible?

### ISSUE THREAD

**Issue by th3fallen** [parcel-bundler/parcel#1032]
I'm trying to migrate from webpack to parcel but in webpack i had my scss builder to add a resource.scss file to the head of every scss file in my project for variables/mixin's ect.

is that possible with parcel and im just missing it?

```
{
            loader: 'sass-loader',
            options: {
              data: '@import "styles/resources";',
              includePaths: [path.join(cwd, 'app')],
            },
          },
```

**Comment by steffenmllr** [comment]
Same question here. That keeps me from migrating at the moment.

**Comment by steffenmllr** [comment]
I'm using a `.sassrc.js` at the moment which works for `includePaths` but not for `data`:

```js
const path = require('path');
const cwd = process.cwd();

module.exports = {
    data: '@import "_globals";',
    includePaths: [
        path.resolve(cwd, 'node_modules/susy/sass'),
        path.resolve(cwd, 'node_modules/normalize.scss'),
        path.resolve(cwd, 'node_modules'),
        path.resolve(cwd, 'static', 'scss')
    ]
};
```

**Comment by devongovett** [comment]
Fixed by #1035.

**Comment by th3fallen** [comment]
@steffenmllr how are you including that js file? just in your index?
